site/_posts/2017-09-29-miqldap-to-sssd.md

```diff
@@ -164,28 +165,32 @@ Screenshot showing the appliance configuration for Mode: LDAP:
 
 The miqldap_to_sssd conversion is **not reversible**.
 
-It is recommended that your ManageIQ appliance be backed up and/or a snapshot be made of the virtual machine prior to running the *miqldap_to_sssd* conversion tool.
+It is recommended that your ManageIQ appliance be backed up and/or a snapshot be made of the virtual machine prior to
+running the *miqldap_to_sssd* conversion tool.
 
 **Run the conversion**
 ---------------------------------------------------------------------
 
-To run the conversion from this *MiqLdap* configuration the *miqldap_to_sssd* command needs to be passed the **--basedn-domain** argument.
-This is because the **Base DN** is not available in the appliance configuration.
+To run the conversion from this *MiqLdap* configuration the *miqldap_to_sssd* command needs to be passed
+the **basedn-domain**, **bind-dn** and **bind-pwd** arguments.
+This is because, with this configuration, values for these items are not available in the appliance database.
 
-*miqldap_to_sssd* supports 5 arguments. This example uses the **--basedn-domain** argument.
+*miqldap_to_sssd* supports 7 arguments. This example uses the **basedn-domain**, **bind-dn** and **bind-pwd** arguments.
 <pre><code style="white-space: pre">
 miqldap_to_sssd --help
 Usage: ruby tools/miqldap_to_sssd.rb [options]
-  -d, --basedn-domain=                       The Base DN domain name, e.g. example.com
-  -c, --tls-cacert=                          Path to certificate file
+  -d, --basedn-domain                        The Base DN domain name, e.g. example.com
+  -b, --bind-dn                              The Bind DN, credential to use to authenticate against LDAP e.g. cn=Manager,dc=example,dc=com
+  -p, --bind-pwd                             The Base DN domain name, e.g. example.com
```

Comment:
The description doesn't seem to match the argument.

Reply:
@cybette Thank you for the reply.  I don't quite understand you comment
tls-cacert is the argument where the certificate file path can be specified.

Do you want me to change the name of the flag to include `file`?

e.g.: --tls-cacert-file

Reply:
@jvlcek I'm referring to `bind-pwd` with the description "The Base DN domain name, e.g. example.com"

Reply:
@cybette AH! Yes. I see it now! Thank you. I will fix it.